File: DigitalRSVP.App/src/app/data/event.ts
export class Event {
    private _id: string = '';
    private _name: string = '';
    private _expiryDate: Date | null = null;

    constructor() {}

    get id(): string {
        return this._id;
    }
    set id(value: string) {
        this._id = value;
    }

    get name(): string {
        return this._name;
    }
    set name(value: string) {
        this._name = value;
    }

    get expiryDate(): Date | null {
        return this._expiryDate;
    }
    set expiryDate(value: Date) {
        this._expiryDate = value;
    }
}

File: DigitalRSVP.Core/Models/Event.cs
namespace DigitalRSVP.Core.Models
{
    public class Event
    {
        private Guid _id;
        public Guid Id
        {
            get
            {
                return this._id;
            }
            set
            {
                this._id = value;
            }
        }

        private string _name;
        public string Name
        {
            get
            {
                return this._name;
            }
            set
            {
                this._name = value;
            }
        }

        //private string _contactEmail;
        //public string ContactEmail
        //{
        //    get
        //    {
        //        return this._contactEmail;
        //    }
        //    set
        //    {
        //        this._contactEmail = value;
        //    }
        //}

        private DateTime _expiryDate;
        public DateTime ExpiryDate
        {
            get
            {
                return this._expiryDate;
            }
            set
            {
                this._expiryDate = value;
            }
        }
    }
}
